client/src/MainLayout.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './MainLayout.css';
+
+const MainLayout: React.FC = () => {
+    return (
+        <div className="main-layout">
+            <header className="layout-header">
+                <h1>cat-GPT</h1>
+            </header>
+            <main className='chat-sec'>
+                <div>
+                    {/* Chat Area Placeholder */}
+                    <span>CHAT AREA</span>
```

Comment:
Can you place a dummy Cat GIF !!

Reply:
![catGPT](https://github.com/Sweetdevil144/Cat-GPT/assets/146512765/4fc86346-d573-4df7-86b7-fec70c4a55f1)
is this good?